impeller/docs/faq.md

```diff
@@ -100,3 +100,113 @@
+  * To make this process self-serve, [a scheme was
+    devised](https://github.com/flutter/flutter/issues/53607) to run the
+    application and its tests during development in training phase, capture the
+    shaders, package them with the application, and during a new "warm up"
+    phase,  pre-compile the shaders.
+  * But, remember these shaders were platform specific? This moved the burden of
+    platform specfic warmup to the developers. It also made the application
+    sizes a lot bigger since the shaders now had to be packaged with the
+    application. Application launch times were delayed as well.
+  * Even developers who went through these steps weren't always guaranteed
+    freedom from shader compilation jank. If some screens were missed during
+    training runs due to incomplete automated or manual testing, jank could be
+    expected. For complex applications where certain flows were blocked behind
+    authentication or other restrictions, this was an easy situation to get
+    into. Also, some animation frames could be skipped during training because
+    jank during previous frames in the training run. So training runs not only
+    had to be run over the entire app, but also multiple times (on all
+    platforms) to ensure success. Moreover, applications that were really
+    effective at training began to run into egregiously high first-frame times
+    ([some as high as 6
+    seconds](http://github.com/flutter/flutter/issues/97884)).
+  * Due to these reasons, developers abandoned this self-serve warmup mechanism.
+    For instance, no application in Google used this due to the all the problems
+    mentioned.
+  * Flutter began to get a reputation as being hard to write performant apps
+    with. The sort of consistent performance that could be expected from
+    frameworks like UIKit on iOS. Sadly, this was true. And not good enough! We
+    needed to do better.
+  * Impeller was borne out the experience from these events. There were other
+    performance issues the team couldn't address with the previous renderer.
+    But, in truth, the mandate to work on Impeller would never have materialized
+    if it weren't for the issue of shader compilation jank. This was in-spite of
+    the massive sunk cost of attempting all the workarounds.
+  * A very early look at how Impeller avoids the problems in the old renderer
+    are [discussed in this
+    tech-talk](https://www.youtube.com/watch?v=gKrYWC_SDxQ). The renderer is way
+    more advanced today but that talk describes the very early thinking,
+    motivations, and decisions.
+  * With Impeller shipping on multiple apps, the open issues related to shader
+    compilation jank have been fixed and closed.
+  * Today, Impeller outperforms the old renderer not only in worst-frame timed
+    benchmarks, but is also faster on average.
+* What makes Impeller different, compared to Skia, for Flutter?
+  * All of Impellers shaders are [authored and
```

Comment:
authored could be interpreted as "manually authored". Is there another term? Generated? Created?

maybe I'm overthinking it.

Reply:
Done.